x-pack/plugins/maps/public/meta.js

```diff
@@ -95,7 +98,7 @@ export function getEMSClient() {
 
 export function getGlyphUrl() {
```

Comment:
since we're fixing, can you add a unit test for `getGlyphUrl` for the expected behavior:

- EMS disabled -> use local fonts
- EMS enabled, not proxied through kibana: -> use fontlibry-url
- EMS enabled, proxied through kibana -> use proxy-path